### DISCUSSION TITLE
Lodestar Standup 2024-45 (December 17, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7290] [Standups]
### Meeting Info
[December 17, 2024, 15:00-16:00 UTC](https://savvytime.com/converter/utc-to-pst-est-cet-ist-ict/dec-17-2024/3pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 15:00 UTC

### Agenda
- Open Discussion
  - TBD
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
it happened multiple times that it takes time to find the issue that cause performance regression due to 30 day window limit of metrics. Is it possible to extend that to 60 days, need to figure out the cost as well. It doesn't have to be for all hosts, only stable + unstable if it's expensive I guess

**Reply by philknows** [reply]
Already working with @Faithtosin to extend metrics collection window to 60 days.

**Comment by nflaig** [comment]
Should consider including https://github.com/ChainSafe/lodestar/pull/7304 in our next release (the one after 1.24)

**Comment by philknows** [comment]
Closing for #7326
